src/component/caculationWinner.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+function calculateWinner(squares: string[]) {
```

Comment:
Look like utility method, should belong to `ts` file. We don’t serve any components here, don’t need to import React or put those code inside `tsx` file